test/integration/targets/openssl_csr/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+- import_role:
+    name: setup_openssl
+
+- name: Generate privatekey
+  openssl_privatekey:
+    path: '{{ output_dir }}/privatekey.pem'
+
+- name: Generate CSR
+  openssl_csr:
+    path: '{{ output_dir }}/csr.csr'
+    privatekey_path: '{{ output_dir }}/privatekey.pem'
+    commonName: 'www.ansible.com'
+
+- import_tasks: ../tests/validate.yml
```

Comment:
@pilou- 

> why import_tasks: ../tests/validate.yml instead of import_tasks: validate.yml?

Because for some reason it can't find the proper path to the file if not specified that way.

```
#> source hacking/env-setup
#> ansible-test integration --docker=centos7 -v openssl_csr
```

> why are validation tasks split in another file (validate.yml) and not kept in main.yml?

To logically separate what is the scenario I am testing vs. the test suite to validate the scenario.

If this is a blocker I could revert, but if possible, I'd like to keep it that way